dismorfo/ultraviolet-nightwatch
# Invenio58

This PR adds the following:

- environment variables for username and password
- login before action command

To test this PR:
- ensure your ultraviolet repository is up to date on development branch and running (`invenio-cli services start`, `invenio-cli run`)
- run `yarn run test nightwatch/tests/features/INVENIO-58.js`
